src/storage-types/src/sources/mysql.rs

```diff
@@ -135,6 +135,36 @@ pub static MYSQL_PROGRESS_DESC: Lazy<RelationDesc> = Lazy::new(|| {
         .with_column("transaction_id", ScalarType::UInt64.nullable(true))
 });
 
+impl MySqlSourceConnection {
```

Comment:
Class not covered by any test according to the [coverage report](https://buildkite.com/materialize/coverage/builds/413)?
Or should this be covered by the MySQL tests that I added?

Reply:
I inadvertently lost the diff that contained your tests; recovered it and they're back in this PR.